### ISSUE TITLE
Dimmer Cache Issue: Brightness falls back to 100%

### ISSUE THREAD

**Issue by pgerke** [pgerke/homebridge-freeathome-local-api#7]
Alright, tried it out, and it does work now! I'm facing one other issue but maybe we can create a separate issue:

Whenever I change the brightness of one of my dimmer lights, it says in the logs that it's changing the brightness to that value but immediately after it also changes it to 100%, hence my lights almost always go to full brightness...

_Originally posted by @rubenstolk in https://github.com/pgerke/homebridge-freeathome-local-api/issues/6#issuecomment-1158640605_

**Comment by pgerke** [comment]
I have pushed [v0.2.3](https://www.npmjs.com/package/homebridge-freeathome-local-api/v/0.2.3-pre) to npm which should fix the issue, but please note that the stability test is pending for the new version.

**Comment by rubenstolk** [comment]
Thanks! Will give it a try this week!

**Comment by rubenstolk** [comment]
Yep, the issue is resolved, and amazingly I can now control the lights without any delay (unlike @superyaro plugin which is sadly not being developed anymore).

Let me know how I can support the development of this plugin!

**Comment by pgerke** [comment]
Happy to hear that! Please feel free to bring up any further issues you encounter!
